src/ui/utils/common.ts

```diff
@@ -13,3 +13,11 @@ export function getImageDimension(file: File): Promise<{
     img.src = URL.createObjectURL(file);
   });
 }
+
+export function bytesToSize(bytes: number) {
```

Comment:
Thanks for adding this! In my first upload I almost uploaded the wrong file but caught it because the file size was wrong. Really great feature 👍